cardInfoAPI-Level1.yaml

```
@@ -0,0 +1,1402 @@
+        - in: query
+          name: clearing_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/ClearingReference'
+          description: >
+            Retrieves the transaction related to an individual clearing.
+            Query parameter only returns results for cleared transactions (those booked on the card account for credit and prepaid or bank account for debit cards).
+        - in: query
+          name: card_id
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardId'
+          description: Retrieves all transactions related to a card. Value has to be a technical identifier for the card that can be retrieved with the GET /cards endpoint.
+        - in: query
+          name: token_id
+          required: false
+          schema:
+            $ref: '#/components/schemas/TokenId'
+          description: Retrieves all transactions related to a token. Value has to be a technical identifier for the card that can be retrieved with the GET /card-tokens endpoint.
+        - in: query
+          name: from_date
+          required: false
+          schema:
+            $ref: '#/components/schemas/DateTime'
+          description: Retrieves all transactions that took place after the provided date time. Filtering is applied on the transaction_date_time attribute.
+        - in: query
+          name: to_date
+          required: false
+          schema:
+            $ref: '#/components/schemas/DateTime'
+          description: Retrieves all transactions that took place before the provided date time. Filtering is applied on the transaction_date_time attribute.
+        - in: query
+          name: status
+          required: false
+          schema:
+            $ref: '#/components/schemas/TransactionStatus'
+          description: Retrieves all transactions in a certain status. Only one status can be filtered at a time.
+        - in: query
+          name: original_currency
+          required: false
+          schema:
+            $ref: '#/components/schemas/OriginalCurrency'
+          description: Retrieves all transactions in a certain currency. Filtering is applied on the original_currency attribute.
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Transactions were found which match the query parameters. If no transactions matched the parameters or access rights are missing, an empty list is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: array
```

Array as top level is not extendable e.g., for pagination.

```suggestion
                type: object
                properties:
                  cardTransations:
                    type: array
```

---

We will convert it into an object and add pagination for this endpoint.